packages/engine/Source/Scene/I3SDataProvider.js

```diff
@@ -70,6 +75,10 @@ import Rectangle from "../Core/Rectangle.js";
  * @property {ArcGISTiledElevationTerrainProvider|Promise<ArcGISTiledElevationTerrainProvider>} [geoidTiledTerrainProvider] Tiled elevation provider describing an Earth Gravitational Model. If defined, geometry will be shifted based on the offsets given by this provider. Required to position I3S data sets with gravity-related height at the correct location.
  * @property {boolean} [traceFetches=false] Debug option. When true, log a message whenever an I3S tile is fetched.
  * @property {Cesium3DTileset.ConstructorOptions} [cesium3dTilesetOptions] Object containing options to pass to an internally created {@link Cesium3DTileset}. See {@link Cesium3DTileset} for list of valid properties. All options can be used with the exception of <code>url</code> and <code>show</code> which are overridden by values from I3SDataProvider.
+ * @property {boolean} [enableFeatureFiltering=false] Determines if the features will be shown.
+ * @property {boolean} [adjustMaterialAlphaMode=false] The option to adjust the alpha mode of the material based on the transparency of the vertex color. When <code>true</code>, the alpha mode of the material (if not defined) will be set to BLEND for geometry with any transparency in the color vertex attribute.
+ * @property {boolean} [applySymbology=false] Determines if the I3S symbology will be parsed and applied for the layers.
```

Comment:
I'm curious why this is an option. I understand it parses and applies styling like outlines. Is there a reaon this would be defined, but the user would not want to use it?

Reply:
This functionality does additional calculations to generate outlines indices by vertex positions. Disabling this option could save the decoding time when the performance is critical.
applySymbology option applies outlines styling if it is defined in I3S data. We use CESIUM_primitive_outline extension for glTF to draw outlines. This extension requires indices for all outlines lines. I3S data doesn’t contain this information. We generate these indices during I3S decoding, and edges detection by vertex positions takes additional time. Also, CESIUM_primitive_outline extension doesn’t support different outlines colors for features of the same Cesium3DTileset, though it may be required by I3S symbology.
We didn’t find another way in Cesium engine for outlines support, so we keep this as option to allow users disable it due to possible performance or visualization issues.

Reply:
OK. Let's add a bit more information here that this will generate outlines if present, which could be a performance hit.